pkg/controller/snapshot/device/controller.go

```diff
@@ -40,6 +42,7 @@ func NewController(mastership mastershipstore.Store, changes changestore.Store,
 		Resolver: &Resolver{
 			snapshots: snapshots,
 		},
+		NodeID: cluster.GetNodeID(),
```

Comment:
NodeID injected that can be set by tests and here we use cluster.GetNodeID to get its value.